### DISCUSSION TITLE
Color Palette

### DISCUSSION THREAD

**Discussion by Erlicht** [FSchwendinger/EffectVisR#2] [Ideas]
Preferred Color Palette should be red for negative, yellow for neutral and green for positive given a traffic light system.
Inclusion of RColorBrewer palettes would be nice, which would give acces to RdYlGn which fits this.

**Comment by Erlicht** [comment]
Also, the frontiers Color Palette from ggsci fits the description nicely. Booth of these options are however not continuous. Using not continuous colors would be less than ideal.
Can we "mix" the colors based on the value without using a color palette? Do we have to define our own color palette?

**Reply by FSchwendinger** [reply]
- [X] Added RdYlGn colour palette to resolve this.
